docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #57.        k = n - m + 1
</pre><p>If we use <code>k</code> as the function parameter then let's call this <code>l = n - m + 1</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
@seanlaw 
Just to confirm, I should use `l` in other functions as well even if I do not have the parameter `k` in those functions (Because, I notice something similar in `_prescrump` function)

Reply:
<p>I think, in general, <code>i</code> and <code>j</code> should be reserved for looping. Then we move to <code>k</code> and <code>l</code>. If <code>k</code> is in use then use <code>l</code> where appropriate. When <code>k</code> is logically available then use that first before <code>l</code>.  This is more art than science :)      </p>

Reply:
> This is more art than science

Yes!

> The biggest concern about using `l` is that it is hard to read since it looks like the number `1` but we'll use it where we can

This is the first thing I noticed. And, I read before that it is better to avoid using `l` as it looks like number 1. However, I know that you are trying to be really careful in naming variables. So, I guessed you must have a very good reason to use `l`. 

I will use `k` as number of subsequences (i.e. `n-m+1`) if, as you said, it is logically available in a function.